Updates to tests including quarantining and tagging to fit the Docker requirements for automated testing. (#4782)
awsdocs/aws-doc-sdk-examples: php/example_code/iam/README.md

* [Create a service-linked role](GettingStartedWithIAM.php#L45) (`CreateServiceLinkedRole`)
* [Create a user](GettingStartedWithIAM.php#L45) (`CreateUser`)
* [Get a policy](GettingStartedWithIAM.php#L45) (`GetPolicy`)
* [Get a role](GettingStartedWithIAM.php#L45) (`GetRole`)
* [Get the account password policy](GettingStartedWithIAM.php#L45) (`GetAccountPasswordPolicy`)
* [List SAML providers](GettingStartedWithIAM.php#L45) (`ListSAMLProviders`)
* [List groups](GettingStartedWithIAM.php#L45) (`ListGroups`)
* [List inline policies for a role](GettingStartedWithIAM.php#L45) (`ListRolePolicies`)
* [List policies](GettingStartedWithIAM.php#L45) (`ListPolicies`)
* [List policies attached to a role](GettingStartedWithIAM.php#L45) (`ListAttachedRolePolicies`)
* [List roles](GettingStartedWithIAM.php#L45) (`ListRoles`)
* [List users](GettingStartedWithIAM.php#L45) (`ListUsers`)

### Scenarios

Code examples that show you how to accomplish a specific task by calling multiple
functions within the same service.

* [Create a user and assume a role](GettingStartedWithIAM.php) 

## Run the examples

### Instructions

<!--custom.instructions.start-->

> Run the example with the command:

Reviewer: ```suggestion
Run the example with the following command:
```

`php GettingStartedWithIAM.php`
<!--custom.instructions.end-->

#### Create a user and assume a role

This example shows you how to create a user and assume a role. 

* Create a user with no permissions.
* Create a role that grants permission to list Amazon S3 buckets for the account.
* Add a policy to let the user assume the role.
* Assume the role and list S3 buckets using temporary credentials, then clean up resources.

<!--custom.scenario_prereqs.iam_Scenario_CreateUserAssumeRole.start-->
<!--custom.scenario_prereqs.iam_Scenario_CreateUserAssumeRole.end-->

<!--custom.scenarios.iam_Scenario_CreateUserAssumeRole.start-->
<!--custom.scenarios.iam_Scenario_CreateUserAssumeRole.end-->

### Tests

⚠ Running tests might result in charges to your AWS account.

To find instructions for running these tests, see the [README](../../README.md#Tests)
in the `php` folder.

<!--custom.tests.start-->

> The tests for IAM follow an older process and are not currently recommended.

Reviewer: ```suggestion
The tests for IAM follow a legacy process and are not currently recommended.
```
